src/Scene/NodeProcess.js

```diff
@@ -402,6 +444,8 @@ NodeProcess.prototype.processNode = function processNode(node, camera, params) {
             this.refineNodeLayers(node, camera, params);
         }
 
+        this.getContentLayers(node, camera, params);
```

Comment:
but content needs to be fetched even if they are hidden. Because content is visible and the same for superior level.